File: 1979. Find Greatest Common Divisor of Array/solution.java
class Solution {
    public int findGCD(int[] nums) {
        Arrays.sort(nums);
        int a,b;
        a=nums[0];
        b=nums[nums.length-1];
        while(true){
            if(a==b){
                return a;
            }
            if(a<b){
                b=b-a;
            }
            if(b<a){
                a=a-b;
            }
        }
    }
}

File: 1979. Find Greatest Common Divisor of Array/solution.py
class Solution:
    def findGCD(self, nums: List[int]) -> int:
        nums.sort()
        a=nums[0]
        b=nums[len(nums)-1]
        while(True):
            if(a==b):
                return a
            
            if(a<b):
                b=b-a
            
            if(b<a):
                a=a-b
            
        